source/developers/projects/craftercms/index.rst

```diff
@@ -762,36 +762,34 @@ What this does under the hood is:
 BUNDLE
 ^^^^^^
 
-The Gradle task ``bundle`` will build a deployable and distributable bundle of Crafter CMS for the authoring and/or delivery environments.  This will generate zip and tar files ready to be unarchived and run.
+The Gradle task ``bundle`` will build deployable and distributable binaries of Crafter CMS for the authoring and/or delivery environments.  This will generate tar files ready to be unarchived and run.
 
     .. code-block:: bash
 
-       ./gradlew bundle [-Penv={env}] [-Pcrafter.bundle.archive={tar|zip}]
+       ./gradlew bundle [-Penv={env}]
 
-Archives will be saved as ``crafter-cms-authoring.tar`` and ``crafter-cms-authoring.zip`` for the *Authoring Environment* and ``crafter-cms-delivery.tar`` and ``crafter-cms-delivery.zip`` for the *Delivery Environment* in the ``bundles`` folder
+Binaries will be saved as ``crafter-cms-authoring-VERSION.tar`` for the *Authoring Environment* and ``crafter-cms-delivery-VERSION.tar`` for the *Delivery Environment* in the ``bundles`` folder
 
-Using the bundle task property ``crafter.bundle.archive`` lets you select what archive (tar or zip) will be generated, and the common task property ``env`` lets you select what environment (authoring or delivery) will be generated.
+Using the common task property ``env`` lets you select what environment (authoring or delivery) will be generated.
 
-Let's look at an example using the two task properties mentioned above:
+Let's look at an example using the two task property mentioned above:
 
     .. code-block:: bash
 
-        ./gradlew bundle -Penv=authoring -Pcrafter.bundle.archive=zip
+        ./gradlew bundle -Penv=authoring
 
-The command above will generate an authoring zip bundle in the bundles folder named ``crafter-cms-authoring.zip``.  If the property ``crafter.bundle.archive`` is not specified, an authoring zip and tar file bundles will be generated.
+The command above will generate an authoring binary archive in the bundles folder named ``crafter-cms-authoring-VERSION.tar.gz``.
 
-There are four more gradle bundle tasks available for use aside from ``bundle``:
+There are two more gradle bundle tasks available for use aside from ``bundle``:
```

Comment:
These tasks are gone in 4, heads up. The new world is just `bundle` and it'll bundle. -PcurrentPlatform=darwin makes it build Mac. I wonder if this stuff belongs here or in GitHub readme for the craftercms project.